ci-operator/step-registry/baremetalds/assisted/operator/setup/before-upgrade/baremetalds-assisted-operator-setup-before-upgrade-commands.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds assisted operator setup before upgrade command ************"
+
+# Fetch packet basic configuration
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/assisted-service.tar.gz"
+
+# shellcheck disable=SC2087
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+
+set -xeo pipefail
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+export -f wrap_if_ipv6 ipversion
+
+REPO_DIR="/home/assisted-service"
+if [ ! -d "\${REPO_DIR}" ]; then
+  mkdir -p "\${REPO_DIR}"
+
+  echo "### Untar assisted-service code..."
+  tar -xzvf /root/assisted-service.tar.gz -C "\${REPO_DIR}"
+fi
+
+cd "\${REPO_DIR}"
+
+echo "### Setup assisted installer..."
+
+images=(${ASSISTED_AGENT_IMAGE} ${ASSISTED_CONTROLLER_IMAGE} ${ASSISTED_INSTALLER_IMAGE})
+
+cat << VARS >> /root/config
+export DISCONNECTED="${DISCONNECTED:-}"
+
+# TODO: remove this and support mirroring an index referenced by digest value
+# https://issues.redhat.com/browse/MGMT-6858
+export INDEX_IMAGE_INSTALL_OVERRIDE="\$(dirname ${INDEX_IMAGE_INSTALL_OVERRIDE})/pipeline:assisted-operator-index-ocm-2.3"
+echo $INDEX_IMAGE_INSTALL_OVERRIDE
```

Comment:
Let's set these:

```
export DISCONNECTED="false"
export INDEX_IMAGE_INSTALL_OVERRIDE=${INDEX_IMAGE_INSTALL_OVERRIDE}
```

This `dirname` stuff has to deal with the DISCONNECTED flow which we are not invoking in the upgrade flow.

Reply:
Done